src/control/server/mgmt_pool_test.go

```diff
@@ -307,16 +331,18 @@ func TestServer_MgmtSvc_PoolCreate(t *testing.T) {
 	missingSB := newTestMgmtSvc(t, log)
 	missingSB.harness.instances[0].(*EngineInstance)._superblock = nil
 	notAP := newTestMgmtSvc(t, log)
+	defaultTierRatios := []float64{0.06, 0.94}
 
 	for name, tc := range map[string]struct {
-		mgmtSvc       *mgmtSvc
-		setupMockDrpc func(_ *mgmtSvc, _ error)
-		targetCount   int
-		memberCount   int
-		req           *mgmtpb.PoolCreateReq
-		drpcRet       *mgmtpb.PoolCreateResp
-		expResp       *mgmtpb.PoolCreateResp
-		expErr        error
+		mgmtSvc        *mgmtSvc
+		setupMockDrpc  func(_ *mgmtSvc, _ error)
+		targetCount    int
+		memberCount    int
+		mdonssdEnabled bool
```

Comment:
It would be better to use bumpy case for the mdOnSSD portion of the name, too.

Reply:
will do if I have to re-push